docs/_sass/_elements.scss

```diff
@@ -476,16 +476,16 @@ pre {
 }
 
 .ui-state-default, .ui-widget-content .ui-state-default, .ui-widget-header .ui-state-default, .ui-button, html .ui-button.ui-state-disabled:hover, html .ui-button.ui-state-disabled:active {
-	border: .3rem solid $sless-light-blue;
+	border: .3rem solid $cybr-blue;
 }
 
 .ui-tabs .ui-tabs-nav li.ui-tabs-active {
-	background-color: $sless-light-blue;
-	color: $sless-blue;
+	background-color: $cybr-blue;
+	color: $cybr-blue;
 }
 
 .ui-state-active a, .ui-state-active a:link, .ui-state-active a:visited {
-	color: $sless-blue;
+	color: $cybr-blue;
```

Comment:
Could we get underlining links on hover too, so that it matches the links outside of tabbed pane?

Reply:
I can underline but apparently I can't get it to change color - but at least I can underline!! 😬